src/shared/components/comment/comment-node.tsx

```diff
@@ -292,22 +292,24 @@ export class CommentNode extends Component<CommentNodeProps, CommentNodeState> {
             mark: this.isCommentNew || this.commentView.comment.distinguished,
           })}
         >
-          <button
+          <div
             className={classNames({
               "ms-2": !this.props.noIndent,
             })}
             onClick={linkEvent(this, this.handleCommentCollapse)}
-            role={"textbox"}
+            onKeyDown={linkEvent(this, this.handleKeyBinds)}
```

Comment:
If this is a button, it doesn't need `onKeyDown` -- it handles `onClick` with a space or an enter.

Reply:
It's role="button" but not a <button> anymore. If I remove the onKeydown I get:

Visible, non-interactive elements with click handlers must have at least one keyboard listener  jsx-a11y/click-events-have-key-events